.travis.yml

```diff
@@ -1,5 +1,3 @@
 language: rust
 rust:
-    - stable
```

Comment:
~~I think for now it would be better to use Box<Future> and to stay on stable. But I think it can be done in the next PR.~~

It should compile on stable, right? Since you are using `Box`.